gatekeeper-3.13.yaml

```diff
@@ -33,16 +36,19 @@ pipeline:
       install -Dm755 ./manager ${{targets.destdir}}/usr/bin/manager
 
 subpackages:
-  - name: gatekeeper-compat
+  - name: ${{package.name}}-compat
     pipeline:
       - runs: |
           mkdir -p ${{targets.subpkgdir}}
           ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+    dependencies:
+      provides:
+        - ${{package.name}}-compat=3.13.999
```

Comment:
This is actually wrong now because `package.name` is `gatekeeper-3.13`.

Same comment for `3.12` above.

Reply:
Done